packages/inventory/src/components/table/__snapshots__/EntityTable.test.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`EntityTable DOM should render correctly - compact 1`] = `
 <table
   aria-label="Host inventory"
   class="ins-c-entity-table pf-c-table pf-m-grid-md pf-m-compact"
-  data-ouia-component-id="OUIA-Generated-Table-20"
+  data-ouia-component-id="OUIA-Generated-Table-22"
```

Comment:
I'm unsure why these change and if this is a valid change. I only pushed the snapshot with the results of the test added here, because I thought it might be a different patternfly version difference or something.

Reply:
Yeah, this is valid. PF generates ids automatically and deterministically, and because you added new test cases, there are now more components rendered in the same “instance” so the global ID pool is now bigger. If you run this test seperately, the number will be 1.

... and that’s one of the reasons I don’t like snapshot tests. 😄

Reply:
:D oh, ok. yeah, the snapshot diffs can get excessive.